### PR TITLE
[webapi] Automate 1 test for Sandbox

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 1, Delete 0
Unit test Platform: Andriod
Unit test result summary: Pass 1, Fail 0, Blocked 0

### CODE REVIEW

**Comment by zqzhang** [comment]
I haven't looked into the scripts in details, but seems this BDD automation involves lots of scripts which make the test more complicated.

Another thinking, are we able to sell the BDD automation method to W3C test community, since this is to automate a W3C manual test?

File: webapi/tct-sandbox-html5-tests/tests.xml [hunk 1]
```diff
@@ -307,5 +294,12 @@
         </description>
       </testcase>
     </set>
+    <set name="sandbox-bdd" type="js" ui-auto="bdd">
+      <testcase component="W3C_HTML5 APIs/Security/HTML5 The iframe element" execution_type="auto" id="attri_default_block_submit" purpose="Block form submission inside iframe with sandbox attribute">
+        <description>
+          <test_script_entry test_script_expected_result="0">/opt/tct-sandbox-html5-tests/bddrunner html5-sandbox-app/testscripts/sandbox_007.feature</test_script_entry>
```
Comment (cicili) [inline webapi/tct-sandbox-html5-tests/tests.xml]: please update this entry follow new format

**Comment by cicili** [comment]
@JianfengXu , please update the w3c html which used in bdd position, follow our new rules(check my mail for details)

**Comment by cicili** [comment]
don't change the manual here, webapi/tct-sandbox-html5-tests/sandbox/w3c/{sandbox_007-manual.htm → bdd/sandbox_007.htm}

**Comment by cicili** [comment]
"Another thinking, are we able to sell the BDD automation method to W3C test community, since this is to automate a W3C manual test?", I think its ok, but BDD set up need some actions/efforts, you can raise this open in our weekly meeting to discuss it

**Comment by cicili** [comment]
LGTM

**Pull request merged**